### DISCUSSION TITLE
Questions about rekey_time

### DISCUSSION THREAD

**Discussion by FrontWhite** [strongswan/strongswan#1847] [Q&A]
Hello,When I configure rekey_time, the value is set to 600 seconds. However, after the expiration of rekey, both sides of the tunnel ping failed, and it takes a long time to recover. I have a question: Is there a minimum value of rekey_time? If not, how do I solve this problem?

**Comment by strongX509** [comment]
Without a strongSwan log file I cannot diagnose your problem. A 10 minute rekey time might be ok for some test setup in a production system it seems rather short for me.
